.github/workflows/nparse-build.yml

```diff
@@ -0,0 +1,46 @@
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+# https://docs.github.com/en/actions/using-workflows/about-workflows
+
+name: nParse Build
+
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    # Set the target Platform
+    runs-on: windows-latest
+
+    steps:
+    # Checkout the code (See Syntax: https://github.com/actions/checkout)
+    - uses: actions/checkout@v4
+    # Setup Python (See Syntax: https://github.com/actions/setup-python)
+    - name: Set up Python 3.12
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.12"
+    # Update pip and install requirements
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt pyinstaller
+    # Build the exe with PyInstaller
+    - name: Build the application with PyInstaller
+      run: |
+        PyInstaller nparse_py.spec
+    # Prepair assets for packaging
+    - name: Prepair assets for packaging
+      run: |
+        move data dist
+        Compress-Archive -Path .\dist\* -Destination nparse_${{ github.ref_name }}.zip
+    # Create Release (See Syntax: https://github.com/softprops/action-gh-release)
+    - name: Release
+      uses: softprops/action-gh-release@v2
+      with:
+        generate_release_notes: True # This auto generates release notes. (See Syntax: https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes)
```

Comment:
Neat! Is there an example of what the source for this looks like?

Reply:
I found relevant info at: https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

The release.yml file is like a template sort of for the release notes from what I could gather. I tried to setup a defaultish template based on the tags that were currently setup/being used in the issues.

I believe the behavior should be the same behavior as if you edit a previous release, and in the notes, hit the auto generate release notes button thingy I believe, not positive though.

On my test branch it simply added a changelog dif between the versions, but I also didn't have any PR's or anything in the test branch I used to create it.

We should definitely look into customizing the release.yml more for sure.